src/datasets/inspect.py

```diff
@@ -99,7 +107,18 @@ def inspect_metric(path: str, local_path: str, download_config: Optional[Downloa
     )
 
 
-def get_dataset_infos(path: str):
+def get_dataset_infos(
+    path: str,
+    revision: Optional[Union[str, Version]] = None,
+    download_config: Optional[DownloadConfig] = None,
+    download_mode: Optional[GenerateMode] = None,
+    force_local_path: Optional[str] = None,
+    dynamic_modules_path: Optional[str] = None,
+    return_resolved_file_path: bool = False,
+    return_associated_base_path: bool = False,
```

Comment:
These parameters may get removed from `prepare_module` in #2986 , so maybe let's not have them here

Reply:
ok

Reply:
Done